src/api/editMessage.js

```diff
@@ -0,0 +1,15 @@
+import { apiPatch } from './apiFetch';
```

Comment:
The reason is that 'edit' is more of a continuous action while update sound more like a command. Makes sense?